search/search_quality/assessment_tool/main_view.cpp

```diff
@@ -43,6 +44,14 @@ MainView::MainView(Framework & framework) : m_framework(framework)
   InitMapWidget();
   InitDocks();
   InitMenuBar();
+
+  m_framework.SetMapSelectionListeners(
+      [this](place_page::Info const & info) {
```

Comment:
Please, look at the whole call - there are two lambdas, and they are located one just below the other.